autopush/logging.py

```diff
@@ -76,7 +76,7 @@ def stdout(message):
     else:
         sys.stdout.write("       %s\n" % message['message'])
     return
-    """
+    # """
```

Comment:
Actually, sorta intentional. It lets me activate the "human readable logging" by adding a # to the start of line 73 instead of to both lines. The triple quote stops on that line regardless of the starting hash.